src/dbally/context/_utils.py

```diff
@@ -0,0 +1,75 @@
+from inspect import isclass
+from typing import Any, Optional, Sequence, Tuple, Type, Union
+
+import typing_extensions as type_ext
+
+from dbally.context.context import BaseCallerContext
+from dbally.views.exposed_functions import MethodParamWithTyping
+
+ContextClass: type_ext.TypeAlias = Optional[Type[BaseCallerContext]]
+
+
+def _extract_params_and_context(
+    filter_method_: type_ext.Callable, hidden_args: Sequence[str]
+) -> Tuple[Sequence[MethodParamWithTyping], ContextClass]:
+    """
+    Processes the MethodsBaseView filter method signauture to extract the args and type hints in the desired format.
```

Comment:
Typos: "signauture", "claases", "are getting excluded the returned"

Reply:
Thanks.